testenv/aws/config.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2024 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"context"
+	"errors"
+
+	envtypes "github.com/openclarity/vmclarity/testenv/types"
+)
+
+const DefaultAWSRegion = "eu-central-1"
```

Comment:
Is this being used somewhere? I see a validation to force providing this env, but wouldn't it be nicer if there were a default region instead of throwing an error?

Reply:
nice catch! I remember I used it initially but then deprecated its use..

Not sure what would be the best practice here, is it is ok for us to force a certain location if the user does not specify it? I think the worst case scenario here would be if the user as all the resources in a certain region and then the stack is created in another region.

Reply:
This test runs on its own bubble meaning it also creates the asset VM it scans so I don't think it matters (or it shouldn't matter) what other resources are in the region besides the ones created by the e2e test, right?

Reply:
that is true, I will add a default region 👍 thanks for reviewing!